discovery/v2.ts

```diff
@@ -1177,15 +1554,852 @@ class DiscoveryV2 extends BaseService {
+          _callback(missingParams);
+          return resolve();
+        }
+        return reject(missingParams);
+      }
+
+      const path = {
+        'project_id': _params.projectId
+      };
+
+      const sdkHeaders = getSdkHeaders(DiscoveryV2.DEFAULT_SERVICE_NAME, 'v2', 'deleteProject');
+
+      const parameters = {
+        options: {
+          url: '/v2/projects/{project_id}',
+          method: 'DELETE',
+          path,
+        },
+        defaultOptions: extend(true, {}, this.baseOptions, {
+          headers: extend(true, sdkHeaders, {
+          }, _params.headers),
+        }),
+      };
+
+      return this.createRequest(parameters).then(
+        res => {
+          if (_callback) {
+            _callback(null, res);
+          }
+          return resolve(res);
+        },
+        err => {
+          if (_callback) {
+            _callback(err)
+            return resolve();
+          }
+          return reject(err);
+        }
+      );
+    });
+  };
+
+  /*************************
+   * userData
+   ************************/
+
+  /**
+   * Delete labeled data.
+   *
+   * Deletes all data associated with a specified customer ID. The method has no effect if no data is associated with
+   * the customer ID.
+   *
+   * You associate a customer ID with data by passing the **X-Watson-Metadata** header with a request that passes data.
+   * For more information about personal data and customer IDs, see [Information
+   * security](https://cloud.ibm.com/docs/discovery-data?topic=discovery-data-information-security#information-security).
+   *
+   *
+   * **Note:** This method is only supported on IBM Cloud instances of Discovery.
+   *
+   * @param {Object} params - The parameters to send to the service.
+   * @param {string} params.customerId - The customer ID for which all data is to be deleted.
+   * @param {OutgoingHttpHeaders} [params.headers] - Custom request headers
+   * @param {Function} [callback] - The callback that handles the response
+   * @returns {Promise<DiscoveryV2.Response<DiscoveryV2.Empty>>}
+   */
+  public deleteUserData(params: DiscoveryV2.DeleteUserDataParams, callback?: DiscoveryV2.Callback<DiscoveryV2.Empty>): Promise<DiscoveryV2.Response<DiscoveryV2.Empty>> {
+    const _params = extend({}, params);
+    const _callback = callback;
+    const requiredParams = ['customerId'];
+
+    return new Promise((resolve, reject) => {
+      const missingParams = getMissingParams(_params, requiredParams);
+      if (missingParams) {
+        if (_callback) {
+          _callback(missingParams);
+          return resolve();
+        }
+        return reject(missingParams);
+      }
+
+      const query = {
+        'customer_id': _params.customerId
+      };
+
+      const sdkHeaders = getSdkHeaders(DiscoveryV2.DEFAULT_SERVICE_NAME, 'v2', 'deleteUserData');
+
+      const parameters = {
+        options: {
+          url: '/v2/user_data',
+          method: 'DELETE',
+          qs: query,
```

Comment:
It looks like a ton of methods were added to this service but you only documented two of them in your commits. I don't think you need to change that at this point but in the future, you should document all of them